### PR TITLE
Fix path to favicons

### PR DESCRIPTION
**Pull request by jaylinski**
As noted in https://github.com/mozilla/http-observatory/issues/419, the favicon doesn't show up.
This probably happend while migrating to webpack.

Also minified existing favicons with tinypng.com.

Fixes mozilla/http-observatory/issues/419

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -27,21 +27,14 @@ plugins = [
   new CopyWebpackPlugin([
     {
       from: 'src/images',
-      to: 'images/',
-      flatten: true
```
Comment (jaylinski) [inline config/webpack.config.js:31]: The default is `flatten: false`: https://webpack.js.org/plugins/copy-webpack-plugin/#patterns

**Comment by jaylinski** [comment]
@april Is it ok to include the built files from `dist`? Or should I remove them?

**Comment by april** [comment]
No dist, please. Thanks.  :)

**Comment by jaylinski** [comment]
@april I removed the `dist` files from the commit. :+1:

**Comment by jaylinski** [comment]
@april Is there anything else I have to do to get this merged?

**Comment by april** [comment]
This is deployed to the live site now, sorry I've been absolutely swamped with other work.  :)

**Pull request merged**